### PR TITLE
Add blog post "Metafacture-core release 5.5.0"

### PR DESCRIPTION
**Pull request by dr0i**
See https://github.com/metafacture/metafacture-core/issues/476.

### CODE REVIEW

File: content/blog/metafacture-core-5.5.0/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: Metafacture-core release 5.5.0
+date: "2023-03-04T14:36:03.2300Z"
+description: "This is about changes coming with the metafacture-core release 5.5.0"
+authors: [{lastname: "Christoph",
+           firstname: "Pascal"}]
+---
+
+*Preamble*
+
+This post describes the new developments in `metafacture-core release 5.4.0` since the [release
```
Comment (TobiasNx) [inline content/blog/metafacture-core-5.5.0/index.md]: 5.5.0

**Review by TobiasNx — CHANGES_REQUESTED** [review]
Otherwise +1

File: content/blog/metafacture-core-5.5.0/index.md (hunk 1 above)
Comment (dr0i) [reply content/blog/metafacture-core-5.5.0/index.md]: thx!

**Comment by dr0i** [comment]
Deployed at https://blog.metafacture.org/metafacture-core-5.5.0/ via github pages.

**Pull request merged**